src/commands/deployment/onboard.ts

```diff
@@ -319,6 +316,6 @@ export const setConfiguration = (
     logger.error(
       `Unable to set storage account and table names in configuration file. \n ${err}`
     );
-    throw err;
+    return false;
```

Comment:
@samiyaakhtar how will the integration test interact with this now. If previously we threw an error will we check for a boolean result?

Reply:
The integration test is not trying to test whether config was written back to file or not (which is what this function is doing), since the onboard command is meant to be ran before a user has setup their config file. Failing the execution just because they don't have an existing config file isn't a good experience. So the integration test isn't gonna verify if writing to config succeeded because there is no init file setup for this integration test

Reply:
@andrebriggs it will however leave a logger message but not fail the test

Reply:
OK, I suspect we will rework the onboard command very soon